File: src/test/java/quick/sirens/models/StartEndTimeTest.java
package quick.sirens.models;



import static org.junit.Assert.assertEquals;

import org.junit.Before;
import org.junit.Test;
import sirens.models.StartEndTime;

public class StartEndTimeTest {
    static StartEndTime startEndTime;

    @Before
    public void init() {
        startEndTime = new StartEndTime(0, 10, 5);
    }

    @Test
    public void getStart() {
        assertEquals(0, startEndTime.getStart());
    }

    @Test
    public void getEnd() {
        assertEquals(10, startEndTime.getEnd());
    }
    
    @Test
    public void getNetworkingStart() {
    	assertEquals(5, startEndTime.getNetworkingStart());
    }

}

File: src/test/java/quick/sirens/models/StartEndTimeTest.scala
package quick.sirens.models

import org.junit.Test
import sirens.models.StartEndTime

class StartEndTimeTest {
  @Test
  def getStart(): Unit = {
    val startEndTime = new StartEndTime(0, 10, 5)

    assert(0 == startEndTime.getStart)
  }

  @Test
  def getEnd(): Unit = {
    val startEndTime = new StartEndTime(0, 10, 5)

    assert(10 == startEndTime.getEnd)
  }

  @Test
  def getNetworkingStart(): Unit = {
    val startEndTime = new StartEndTime(0, 10, 5)

    assert(5 == startEndTime.getNetworkingStart)
  }
}
